### PR TITLE
Update prisma schema

### PR DESCRIPTION
**Pull request by Briantam0422**
Try my first pull request. I have updated the Prisma schema

1. Add more columns to the model Post like description, date, time, and location.
2. add a model PostParticipants. One post can have more than one participants

### CODE REVIEW

**Review by mpark4656 — APPROVED** [review]
Looks good.

File: prisma/schema.prisma [hunk 1]
```diff
@@ -14,17 +14,51 @@ datasource db {
 }
 
 model Post {
```
Comment (ialej001) [inline prisma/schema.prisma:16]: Let's change `Post` to `Event` at all locations in the schema. I believe this is supposed to be the model for an Event so let's make it consistent and appropriate.

File: prisma/schema.prisma [hunk 2]
```diff
@@ -14,17 +14,51 @@ datasource db {
...
-    id        Int      @id @default(autoincrement())
-    name      String
+    id              Int        @id @default(autoincrement())
+    name            String // event title
+    description     String     @db.Text
+    dateTime        DateTime
+    image           String?
+    price           Float      @default(0)
+    maxParticipants Int?       @default(0)
+    invitLink       String?
```
Comment (ialej001) [inline prisma/schema.prisma:24]: typo? misisng an 'e' in `invitLink`

File: prisma/schema.prisma [hunk 3]
```diff
@@ -14,17 +14,51 @@ datasource db {
...
+    address         String
```
Comment (ialej001) [inline prisma/schema.prisma:25]: I believe we were also going to be allowing users to host virtual events? Wouldn't this along with `lat` and `lng` then be nullable? We can treat null as a virtual event in the front end.

**Review by ialej001 — CHANGES_REQUESTED** [review]
Just a few comments on the schema. Generally, I think we can move away from the boilerplate's `Post` and change all mentions of "post" to "event". Our users won't be creating `Post`s but `Event`s instead, right?

Otherwise, I think this is a great starting point for the application schema! Great job

**Comment by Briantam0422** [comment]
@ialej001 I think you are right! It makes more sense now. I've updated the code based on your suggestion. Thanks

**Review by ialej001 — APPROVED** [review]

**Pull request merged**